src/index.tsx

```diff
@@ -51,8 +54,45 @@ if (apiExplorer) {
 
 initializeIcons();
 
-const currentTheme = readTheme();
-loadGETheme(currentTheme);
+let currentTheme = readTheme() || 'light';
+function setCurrentSystemTheme(): void {
+  const themeFromLocalStorage = readTheme();
+
+  if (themeFromLocalStorage) {
+    currentTheme = themeFromLocalStorage;
+  } else {
+    currentTheme = checkTheme();
```

Comment:
This can be renamed to `getOSTheme` so that the function name matches the work of the function

Reply:
Okay. Let me rename that